assets/src/edit-story/components/library/panes/shared/pillGroup/pillGroup.js

```diff
@@ -18,24 +18,30 @@
  * External dependencies
  */
 import styled from 'styled-components';
-import { useRef, useState } from 'react';
+import { useCallback, useRef, useState } from 'react';
 import PropTypes from 'prop-types';
-import { rgba } from 'polished';
 import { v4 as uuidv4 } from 'uuid';
 import { __ } from '@web-stories-wp/i18n';
 
 /**
  * Internal dependencies
  */
-import { ArrowDown } from '../../../../button';
-import { useKeyDownEffect } from '../../../../../../design-system';
+import {
+  Button,
+  BUTTON_SIZES,
+  BUTTON_TYPES,
+  BUTTON_VARIANTS,
+  Icons,
+  Pill,
+  useKeyDownEffect,
+} from '../../../../../../design-system';
 import useRovingTabIndex from '../../../../../utils/useRovingTabIndex';
-import Pill from './pill';
 
 /**
  * Internal dependencies
  */
 import {
```

Comment:
Looks like this file has `Internal dependencies` twice -- maybe let's combine these?